felix/iptables/table.go

```diff
@@ -540,14 +540,25 @@ func (t *Table) UpdateChain(chain *generictables.Chain) {
 	t.logCxt.WithField("chainName", chain.Name).Debug("Adding chain to available set.")
 	oldNumRules := 0
 
+	if chain.ForceProgramming {
```

Comment:
nit: slightly better for this block to come after the following comment "Incref any newly-..."